pbjam/plotting.py

```diff
@@ -217,8 +250,8 @@ def plot_spectrum(self, pg=None, path=None, ID=None, savefig=False):
                 y += 0.8 * h * np.exp(-0.5 * (freq[i] - f)**2 / freq_sigma[i]**2)
             ax.fill_between(f, y, alpha=0.3, facecolor='navy', edgecolor='none',
                             label=r'$\propto P(\nu_{\ell=0})$')
-
-            xlim = [min(freq)-dnu, max(freq)+dnu]
+           
+            xlim = [numax-5*dnu, numax+5*dnu]
```

Comment:
Why hard code in the factor 5?  The previous version looks better as it is more dynamic.  What were the reasons for making this change?

Reply:
The previous version was what made the KDE plots zoom out all the way. 

If you don't manually narrow down the frequency range of the input spectrum, it'll just take 0 to Nyquist. This happens when you use `pbjam.session`. 

The factor of 5 is subjective. It's what's being used when plotting.plot_spectrum() is called from `pbjam.star`. It's only changed once the `norders` is set during the `pbjam.asy_peakbag` call. Maybe there's a way to set it dynamically, but 5 seems to work for the majority of cases.